app/static/src/app/store/fitData/getters.ts

```diff
@@ -54,5 +55,19 @@ export const getters: FitDataGetters & GetterTree<FitDataState, FitState> = {
             };
         }
         return result;
+    },
+
+    [FitDataGetter.allData]: (state: FitDataState): null | AllFitData => {
```

Comment:
I think this getter might merit a comment about why it's needed, and the logic behind returning an empty result if no data and timeVariable, but ok to return with no linkedVariables to show unlinked data on the plot.